controllers/cart.js

```diff
@@ -0,0 +1,64 @@
+const { products, carts, orders } = require('../storage');
+const crypto = require('crypto');
+
+function renderCart(id) {
```

Comment:
I would suggest to rename this function to `createCart`. Also you can get rid of `else `statement 
```
 if (!carts.find((item) => item.userId === id)) {
    const cart = {
      id: crypto.randomUUID(),
      userId: id,
      products: [],
    }
    carts.push(cart);
  }
```